code/api/src/modules/product/query.test.js

```diff
@@ -0,0 +1,81 @@
+import request from 'supertest';
+import express from 'express';
+import graphqlHTTP from 'express-graphql'
+import schema from '../../setup/schema'
+
+describe("product queries", () => {
+  let server;
+
+  beforeAll(() => {
+    server = express();
+    server.use(
+      '/',
+      graphqlHTTP({
+        schema: schema,
+        graphiql: false
+      })
+    );
+  });
```

Comment:
Currently the test file is expecting crate_test to be populated with products. We should probably have a setup and teardown to add/clear out products each time.

Reply:
update: issue resolved